cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgreSQLSink.java

```diff
@@ -370,7 +370,7 @@ private void persistAggregation(NGSIGenericAggregator aggregator) throws CygnusP
         String fieldsForCreate = aggregator.getFieldsForCreate();
         String fieldsForInsert = aggregator.getFieldsForInsert();
         String valuesForInsert = aggregator.getValuesForInsert();
-        String schemaName = aggregator.getTableName(enableLowercase);
+        String schemaName = aggregator.getDbName(enableLowercase);
```

Comment:
If this is a bugfix for a bug existing in last version (1.18.0), then the fix should be mentioned in CHANGES_NEXT_RELEASE.

Reply:
Fixed in [db92f3a](https://github.com/telefonicaid/fiware-cygnus/pull/1819/commits/db92f3a85f3761f2112f958ce6bc426c27dece3b)